cmake/OpenCVFindFlatBuffers.cmake

```diff
@@ -0,0 +1,15 @@
+set(HAVE_FLATBUFFERS FALSE)
+
+if(NOT WITH_FLATBUFFERS)
+  return()
+endif()
+
+list(APPEND CUSTOM_STATUS flatbuffers)
+
+find_package(flatbuffers QUIET)
+if(flatbuffers_FOUND)
+  set(HAVE_FLATBUFFERS 1)
+  list(APPEND CUSTOM_STATUS_flatbuffers "    FlatBuffers:" "${flatbuffers_VERSION}")
+else()
+  list(APPEND CUSTOM_STATUS_flatbuffers "    FlatBuffers:" "NO")
+endif()
```

Comment:
Shall I keep `schema_generated.h` in this PR or it's fine to call the compiler `flatc` at build?